src/java/com/palantir/cassandra/concurrent/LocalReadRunnableTimeoutWatcher.java

```diff
@@ -39,7 +41,12 @@ public class LocalReadRunnableTimeoutWatcher implements Runnable
     private LocalReadRunnableTimeoutWatcher() { }
 
     public void watch(ReadCommand readCommand) {
-        logger.trace("Watching read command {} for timeout {}", readCommand, getTimeout());
+        logger.trace("Watching read command {} {} {} {} for timeout {}",
+                     SafeArg.of("columnFamily", readCommand.getColumnFamilyName()),
+                     UnsafeArg.of("key", readCommand.key),
```

Comment:
I think we won't get a lot of useful information from `ByteBuffer`. What about this?

```suggestion
Hex.bytesToHex(readCommand.key.array())
```

Reply:
makes sense, let me make the change